sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/DoFn.java

```diff
@@ -207,20 +189,16 @@
             String name, CombineFn<AggInputT, ?, AggOutputT> combiner);
   }
 
-  /**
-   * Information accessible when running a {@link DoFn.ProcessElement} method.
-   */
+  /** Information accessible when running a {@link DoFn.ProcessElement} method. */
   public abstract class ProcessContext extends Context {
 
     /**
      * Returns the input element to be processed.
      *
-     * <p>The element will not be changed -- it is safe to cache, etc.
-     * without copying.
+     * <p>The element will not be changed -- it is safe to cache, etc. without copying.
```

Comment:
Oops, reverted autoformat in DoFn. I guess I pressed the autoformat hotkey without selecting a range to format.